diff -aurN pidgin-2.10.9.orig/pidgin.desktop.in.in pidgin-2.10.9/pidgin.desktop.in.in
--- pidgin-2.10.9.orig/pidgin.desktop.in.in	2014-03-09 22:52:43.617076590 +0100
+++ pidgin-2.10.9/pidgin.desktop.in.in	2014-03-09 22:52:54.790839570 +0100
@@ -7,5 +7,5 @@
 StartupNotify=true
 Terminal=false
 Type=Application
-Categories=Network;InstantMessaging;
+Categories=Network;InstantMessaging;GTK;
 @USES_MM_CHAT_SECTION@
